tar/bsdtar.c

```diff
@@ -74,20 +74,13 @@ __FBSDID("$FreeBSD: src/usr.bin/tar/bsdtar.c,v 1.93 2008/11/08 04:43:24 kientzle
  * Per POSIX.1-1988, tar defaults to reading/writing archives to/from
  * the default tape device for the system.  Pick something reasonable here.
  */
```

Comment:
To make sure we don't forget to remove the default tape device someday when libarchive 4.0 does come to pass, I suggest:
```
#if ARCHIVE_VERSION_NUMBER >= 4000000
#undef _PATH_DEFTAPE
#elif defined(__linux)
```

Reply:
I'll follow your advice here since I don't have much context. Thanks for your feedback.